pom.xml

```diff
@@ -78,8 +79,8 @@
     <bouncy.version>1.60</bouncy.version>
 
     <junit.version>4.13.1</junit.version>
-    <jackson.version>2.10.0.pr3</jackson.version>
-    <jackson.databind.version>2.10.5.1</jackson.databind.version>
+    <jackson.version>2.12.3</jackson.version>
+    <jackson.databind.version>2.12.3</jackson.databind.version>
```

Comment:
FYI, but we have two other PRs open to correct this.  I'm going to merge those first.  When that happens, you'll probably get a conflict.  I'll resolve it for you.